### PR TITLE
[AGENT-4489] Implement Airflow Operator for Creating Monitoring Job

### PR DESCRIPTION
**Pull request by oleksandr-saienko**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary
Added BatchMonitoringOperator and MonitoringJobCompleteSensor Airflow operators for Monitoring Job creation and status check

## Rationale
Users should be able to submit Batch Monitoring Jobs using Airflow Operators

### CODE REVIEW

**Comment by oleksandr-saienko** [comment]
Jenkins run

**Comment by oleksandr-saienko** [comment]
Jenkins run

**Comment by oleksandr-saienko** [comment]
Jenkins run

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apache-airflow>=2.3.0
 black>=22.12.0
-datarobot>=3.1.1
+datarobot-early-access>=3.2.0.2023.6.19
```
Comment (elatt) [inline requirements.txt:3]: Can you please pull this out into a dedicated PR so it is easier to track/revert.

**Review by elatt — CHANGES_REQUESTED** [review]
Please try to keep PRs singular in purpose/focus in general. And specifically, since we expect this dependency to just be temporary it definitely deserves its own PR.

File: requirements.txt (hunk 1 above)
Comment (oleksandr-saienko) [reply requirements.txt]: temporary switching to “datarobot-early-access>=3.2.0.2023.6.19” moved to dedicated PR: [AGENT-4596] changed to datarobot-early-acces

**Review by dulcardo — APPROVED** [review]
LGTM

**Comment by oleksandr-saienko** [comment]
> Please try to keep PRs singular in purpose/focus in general. And specifically, since we expect this dependency to just be temporary it definitely deserves its own PR.

Done, separate PR: AGENT-4596

**Review by elatt — APPROVED** [review]
